scripts/poclcc/poclcc.c

```diff
@@ -0,0 +1,179 @@
+#include <stdio.h>
```

Comment:
Yes, please create a new 'bin' folder and remove the obsolete 'scripts' folder. I think we can integrate all functionality of pocl-standalone to the new poclcc binary, right?